lib/ImplicitDiscreteSolve/Project.toml

```
@@ -23,10 +23,14 @@ SymbolicIndexingInterface = "0.3.38"
 Test = "1.10.0"
 UnPack = "1.0.2"
 julia = "1.10"
+JET = "0.9.18"
```

```suggestion
JET = "0.9.18, 0.10"
```
This should support Julia 1.12

---

I thought about that but wouldn't that make it always use JET 0.10, unless JET has a compat restriction for Julia?

---

Keep in mind that JET 0.10 is marketed as experimental, especially for `test_package`

---

This just uses it for typos, so hopefully that would be fine. Anyways, it doesn't work because it needs `Julia 1.12.0-beta1.11`, but `pre` only uses `Julia 1.12.0-beta1`. Very cool.

---

It's `JET's job to provide appropriate compats (and I believe it does so).

> Julia 1.12.0-beta1.11, but pre only uses Julia 1.12.0-beta1. Very cool.

never mind. That sounds like a JET bug...

---

Is Julia 1.12.0-beta1.11 even a thing?

---

no

---

my buest guess is there's a missing comma?

---

It does ask you to use a specific branch at https://github.com/aviatesk/JET.jl/blob/master/CHANGELOG.md#0103: 
"However, to use JET v0.10.3, you need to use Julia version v"1.12.0-beta1.11" or later. As of April 19, 2025, you need to build Julia from https://github.com/JuliaLang/julia/pull/58009 for using this 1.12 beta version."

---

Ok, so probably just a typo there. The real issue is that `test_package` available in JET 0.10.4, even though the changelog says it should be.